DocBook-Cookbook.xpr

```diff
@@ -1,1688 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@der-schmelle2 Did you delete this file by accident? :wink:

Reply:
Hi Tom,
I'm sorry for that.
Today I'm working on a new Windows Laptop with oXygenXML and Gitkraken. Gitkraken told me that I had changed the Docbook-Cookbook.xpr (which was correct). I set this file to ignore and Gitkraken tied to delete it. I don't know why it was deleted from the repo.
Maybe it's a better idea to stash the file.

Peter

Reply:
No problem, don't worry. Such things can happen. :smiley: 

If you want to ignore the file, you can use a "global Git ignore" file like this:

```
$ git config --global core.excludesfile '~/.gitignore'
$ echo DocBook-Cookbook.xpr >> ~/.gitignore
```

Reply:
Hi Tom,
thanks for the PDF output.
As discussed today I'll rewrite the hole section under a new feature branch due to more knowledge I received today. 😊